packages/components/src/components/types/common.types.ts

```diff
@@ -14,3 +14,9 @@ export type TItem = {
 };
 
 export type TTableRowItem = { component: React.ReactNode };
+
+export type TDatePickerOnChangeEvent = {
+    date?: string;
+    duration?: number | null | string;
+    target?: { name?: string; value?: number | string | moment.Moment | null };
```

Comment:
just curious, should we extend React types for event?

Reply:
This was done to match the event type of DateOfBirthPicker with the DatePicker Deriv Component